src/main/java/lotto/output/console/LottoGameViewConsoleOutputPort.java

```diff
@@ -0,0 +1,52 @@
+package lotto.output.console;
+
+import lotto.domain.Lotto;
+import lotto.domain.LottoGrade;
+import lotto.domain.LottoTicket;
+import lotto.output.LottoGameViewOutputPort;
+
+import java.util.List;
+import java.util.Map;
+
+public class LottoGameViewConsoleOutputPort implements LottoGameViewOutputPort {
+
+    @Override
+    public void printLottoCount(int lottoBoughtPrice) {
+        System.out.println(lottoBoughtPrice / Lotto.LOTTO_PRICE + "개를 구매했습니다.");
+    }
+
+    @Override
+    public void printGeneratedLottoList(LottoTicket lottoTicket) {
+        for (List<Integer> lottoNumber : lottoTicket.getLottoNumbers()) {
+            System.out.println(lottoNumber);
+        }
+    }
+
+    @Override
+    public void printLottoResult(Map<LottoGrade, Integer> lottoStatistics) {
+        System.out.println("당첨 통계");
+        System.out.println("---");
+
+        List<LottoGrade> lottoGrades = List.of(
+                LottoGrade.FIFTH,
+                LottoGrade.FOURTH,
+                LottoGrade.THIRD,
+                LottoGrade.SECOND,
+                LottoGrade.FIRST
+        );
+
+        for (LottoGrade lottoGrade : lottoGrades) {
+            System.out.println(makeLottoGradeLabel(lottoGrade, lottoStatistics));
+        }
```

Comment:
여기까지 `stream`으로 처리한다면 욕심일까요 🤣 
```suggestion
        lottoGrades.stream().map(lottoGrade -> makeLottoGradeLabel(lottoGrade, lottoStatistics))
        .forEach(System.out::println);
        
```

Reply:
요기에 스트림을 쓰는건 생각못해봤네요!
고민해보겠습니다